test/test_tracker.cpp

```diff
@@ -565,6 +565,57 @@ TORRENT_TEST(current_tracker)
 	std::printf("done\n");
 }
 
+#if TORRENT_USE_RTC
+TORRENT_TEST(websocket_tracker)
+{
+	int const http_port = start_websocket_server();
```

Comment:
this test fails for me with this python error:
```
Traceback (most recent call last):
  File "../websocket_server.py", line 13, in <module>
    import websockets
ModuleNotFoundError: No module named 'websockets'
```

Reply:
Yes, it requires the `websocket` library, since there is no WebSocket support in the python standard library: https://websockets.readthedocs.io/